src/oas.js

```diff
@@ -101,6 +102,80 @@ function normalizedUrl(oas) {
   return ensureProtocol(url);
 }
 
+function normalizePath(path) {
+  const curlBacketMatch = /{(.*?)}/;
+
+  return path
```

Comment:
How about something like this?

```js
'/users/{user_id}/filter_sets/{filter_set_id}'.replace(/{(.*?)}/g, ':$1')
```

Then we can same a little bit of time on iteration

Reply:
Nice, that's a good improvement!